js_modules/dagit/packages/core/src/pipelines/CompositeSupport.ts

```diff
@@ -2,107 +2,130 @@ import {GraphExplorerSolidHandleFragment} from './types/GraphExplorerSolidHandle
 
 function explodeComposite(
   handles: GraphExplorerSolidHandleFragment[],
-  handle: GraphExplorerSolidHandleFragment,
+  composite: GraphExplorerSolidHandleFragment,
 ) {
-  if (handle.solid.definition.__typename !== 'CompositeSolidDefinition') {
+  if (composite.solid.definition.__typename !== 'CompositeSolidDefinition') {
     throw new Error('explodeComposite takes a composite handle.');
   }
 
-  // Replace all references to this composite's inputs in other solid defitions
-  // with the interior target of the input mappings
-  handle.solid.definition.inputMappings.forEach((inmap) => {
```

Comment:
I left this code unchanged but moved it down in the function (inside the other inputMapping iterator) so it could re-use the same local variables